pkg/framework/aries/api/factory.go

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package api
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/transport"
+)
+
+// TransportProviderFactory allows overriding of aries protocol providers
+type TransportProviderFactory interface {
+	CreateOutboundTransportProvider() transport.OutboundTransport
```

Comment:
I'm a bit confused by this method name: `CreateOutboundTransportProvider`

Does this method create an OutboundTransport or a Factory/Provider of OutboundTransports?

Reply:
suffix is not valid as it creates outbound transport. [Updated](https://github.com/hyperledger/aries-framework-go/compare/463122c87b11e27a303f59b42906628162e71b73..a31807a201229cd5ab417413af54914995b98b87#diff-cbf47775202e3791da3aefcade22c7b8R15).